Dockerfile.mustgather

```diff
@@ -1,10 +1,9 @@
-FROM registry.ci.openshift.org/ocp/4.8:must-gather AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.8 AS builder
+WORKDIR /go/src/github.com/openshift/local-storage-operator/must-gather
```

Comment:
The `/must-gather` on this line is incorrect, given the usage in the second stage.

Reply:
I filed https://github.com/openshift/local-storage-operator/pull/222 to address this. Let's move the conversation there.